htdocs/core/modules/project/mod_project_simple.php

```diff
@@ -150,7 +150,8 @@ public function getNextValue($objsoc, $project)
 		$date = (empty($project->date_c) ? dol_now() : $project->date_c);
 
 		//$yymm = strftime("%y%m",time());
-		$yymm = strftime("%y%m", $date);
+		//$yymm = strftime("%y%m", $date);
+		$yymm = date("%y%m", $date);
```

Comment:
Can you use dol_print_date instead of date ? (date is not clear about timezone behaviour)